src/site/layouts/va_form.drupal.liquid

```diff
@@ -62,15 +62,16 @@
           {% endif %}
         </dl>
 
-        <h2 class="vads-u-margin-top--4"">When to use this form</h3>
 
         {% if fieldVaFormUsage %}
+          <h2 class="vads-u-margin-top--4">When to use this form</h3>
```

Comment:
this was my mistake i think - an h2 that closes with an h3 🙂

Reply:
All good! It was easy to miss tbh! 🎉